tools/tplgtool2.py

```diff
@@ -8,19 +8,25 @@
 import sys
 import typing
 from collections import defaultdict
-from construct import this, Container, ListContainer, Struct, Const, Switch, Select, Array, Bytes, GreedyBytes, GreedyRange, FocusedSeq, Pass, Padded, Padding, Prefixed, String, Flag, Byte, Int16ul, Int32ul, Int64ul, Terminated
+from construct import this, Container, ListContainer, Struct, Const, Switch, Select, Array, Bytes, GreedyBytes, GreedyRange, FocusedSeq, Pass, Padded, Padding, Prefixed, Flag, Byte, Int16ul, Int32ul, Int64ul, Terminated
 from dataclasses import dataclass
 from functools import cached_property, partial
 
-def __enum_dict(enumtype) -> dict:
-    return dict((e.name, e.value) for e in enumtype)
-
-# here is a workaround to make Enum/FlagsEnum accept enum type, like construct 2.9
-def Enum(subcon, enumtype):
-    return construct.Enum(subcon, default=Pass, **__enum_dict(enumtype))
-
-def FlagsEnum(subcon, enumtype):
-    return construct.FlagsEnum(subcon, **__enum_dict(enumtype))
+# Pylint complain about the missing names even in conditional import.
+# pylint: disable=E0611
```

Comment:
Can you move this useful link from the commit message to a comment?
https://github.com/construct/construct/blob/master/docs/transition29.rst

Reply:
I append it after the condition.